wbooks/src/redux/authReducer/reducer.js

```diff
@@ -0,0 +1,23 @@
+import { types } from '../types';
```

Comment:
We specify types for each reducer. Move the types into the `actions.js` file and name the object `actions`